lib/packages/shared-types/events/withdraw-package.ts

```diff
@@ -1,16 +1,29 @@
 import { z } from "zod";
-import { attachmentSchema } from "../attachments";
+import {
+  attachmentSchema,
+  attachmentArraySchemaOptional,
+  attachmentArraySchema,
+} from "../attachments";
 
 // Temporary, will be refactored to an extendable schema with Brian/Mike's back-end
 // work.
 export const withdrawPackageSchema = z.object({
   id: z.string(),
   authority: z.string(),
   origin: z.string(),
-  additionalInformation: z
-    .string()
-    .max(4000, "This field may only be up to 4000 characters.")
-    .optional(),
+  additionalInformation: z.preprocess(
+    (value) => (typeof value === "string" ? value.trimStart() : value),
+    z
+      .string()
+      .max(4000)
+      .refine((value) => value !== "", {
+        message: "Additional Information is required.",
+      })
+      .refine((value) => value.trim().length > 0, {
+        message: "Additional Information can not be only whitespace.",
+      }),
```

Comment:
Shouldn't we use this `refine` anywhere `additionalInformation` is defined?

Reply:
This whole variable/schema actually needs removed. Let me do that now